recipe/meta.yaml

```
@@ -23,22 +23,23 @@ requirements:
     - sphinx 5.0.2
```

```suggestion
    - sphinx
```
Will help with our build-outs